src/app/services/article-list.service.ts

```diff
@@ -0,0 +1,53 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ArticleListService {
+
+
+  postz = [
```

Comment:
to powinna być prywatna zmienna, możesz zrobić funkcję `getAll`, która zwróci `this.postz` i będzie dostępna na zewnątrz czyli publicznie